projects/student-projects/build-1.md

```diff
@@ -5,15 +5,19 @@
 | Time    | Activity                     |
 | ------- | ---------------------------- |
 | 09:45   | Check-in                     |
-| 10:00   | Feedback Survey |
+| 10:00   | [Feedback Survey][feedback-10] |
 | 10:10   | Intro to client-side routing |
-| 10:30   | React routing workshop |
+| 10:30   | [React routing workshop I][routing-1-60] |
+| 11:30   | [React routing workshop I][routing-2-120] |
```

Comment:
Since these are different I'd probably give them different titles
```suggestion
| 10:30   | [DIY React Routing][routing-1-60] |
| 11:30   | [React Router][routing-2-120] |
```